queries.sql

```diff
@@ -31,3 +31,56 @@ WHERE neutered = true;
 SELECT *
 FROM animals
 WHERE  weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Transactions and modifications
+
+BEGIN;
+UPDATE animals SET species = 'undefined';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE animal_name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE animal_name IN ('Charmander', 'Squirtle', 'Blossom', 'Ditto', 'Pikachu');
+COMMIT;
+
+BEGIN;
+DELETE FROM animals WHERE id IN (1,2,3,4,5,6,7,8,9,10,11);
```

Comment:
There's no need for `id IN (1,2,...);` The query will work if you write it like this: 
```
DELETE FROM animals WHERE ;
```